opm/output/eclipse/Summary.cpp

```diff
@@ -375,11 +375,16 @@ quantity fpr( const fn_args& args ) {
 
     const auto& p = args.state.data( "PRESSURE" );
     const auto& pv = args.pv;
-    const auto& sw = args.state.data( "SWAT" );
+    const bool hasSwat = args.state.has( "SWAT" );
+    const auto& swat = hasSwat? args.state.data( "SWAT" ): p;
```

Comment:
Ehhh - if you do not have `SWAT` then you use `PRESSURE` - did that make sense?

Reply:
Is this better?